### ISSUE TITLE
Implement RGB and RGBA functionality and display result 

### ISSUE THREAD

**Issue by jdwilkin4** [DBenMoshe/contrast-ratio-repo#25]
As a user, I should be able to input two RGB values and there should be a contrast ratio result displayed.
As a user, I should also able to input two RGBA values and there should be a contrast ratio result displayed.
We want to mimic the same behavior here:
<img width="880" alt="Screen Shot 2022-08-07 at 5 46 44 PM" src="https://user-images.githubusercontent.com/67210629/183318426-d4b9c643-9a39-4bfe-992e-685d332b067c.png">
https://contrast-ratio.com/

**Comment by AndreyDmitriev39r** [comment]
Should we display relative luminance? I don't know if its super useful info for the user.
In #24 I didn't do that. But it could be done: calculation of luminance is required to get contrast-ratio.

**Comment by jdwilkin4** [comment]
I personally don't think we need to display the relative luminance

**Comment by jdwilkin4** [comment]
We can add RGBA to this too

**Comment by AndreyDmitriev39r** [comment]
Well, I guess I can deal with it, since I've already have some experience with these calculations. 
But I will wait a little bit, maybe somebody else who didn't wrtie JS for our codebase, will take it.
